app/factories/bulkrax/object_factory.rb

```diff
@@ -7,16 +7,16 @@ class ObjectFactory
     include Bulkrax::FileFactory
     define_model_callbacks :save, :create
     class_attribute :system_identifier_field
-    attr_reader :attributes, :object, :unique_identifier, :klass, :replace_files, :update_files
+    attr_reader :attributes, :object, :source_identifier_value, :klass, :replace_files, :update_files
     self.system_identifier_field = Bulkrax.system_identifier_field
```

Comment:
i expected to see a change to this. the idea is that you specify a field in the field mapping and the from on that is the source_identifier and to two on it is the system_identifier

Reply:
I totally understand the 'from' being the source_id, and the 'to' being the system_id. we don't have access to the parser/importer/entry here so it must be passed in to the factory, there isn't really anything I can do to change that...

Reply:
@sephirothkod passing system_identifier into the constructor from the entry seems like a solution. Then we aren’t declaring it at the class level here any more.  maybe rename it work identifier? Or something else not so similar to source_identifier?

Reply:
named it work_identifier throughout